src/App.js

```diff
@@ -31,17 +32,39 @@ function App() {
     setTrickStep(trickStep + 1)
   };
 
-  return (
-      <div className='stacks'>  
-        <Stack stack={ initialStack } /> 
-        <Button value='initial' onClick={ handleClick }/>
-      
-        <Stack stack={ middleStack } />
-        <Button value='middle' onClick={ handleClick }/>
-      
-        <Stack stack={ lastStack } />
-        <Button value='last' onClick={ handleClick }/>
+  const handleReset = (FIRST_TRICK_1_STEP) => {
```

Comment:
Você não precisa desse `FIRST_TRICK_1_STEP`, visto que você já sabe qual é o estado inicial (que exportou de outro arquivo), e esse estado vai ser sempre igual. Então para utilizar o `FIRST_TRICK_1_STEP` global.

```suggestion
  const handleReset = () => {
```